modules/spring-boot-autoconfigure-ext/pom.xml

```diff
@@ -56,6 +56,20 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.ignite</groupId>
+            <artifactId>ignite-spring-data_2.2-ext</artifactId>
```

Comment:
I wonder if we can make this work for all of the supported spring-data versions?
Extensions have different packages, so we shouldn't have any trouble with this.
But if that's too much work I guess the most recent version will do just fine

Reply:
I wrote some thoughts about it in my comment in PR https://github.com/apache/ignite-extensions/pull/41#issue-565370788 
In additional, spring-context 5.2.3 (used in ignite-spring-data_2.2-ext) incompatible with spring-context 5.0.16 (used in ignite-spring-data_2.0-ext) and I should make exclusion it from ignite-spring-data_2.0-ext. But if I do it and write some tests like IgniteAutoconfigureTest this test will be failed with NoSuchMethodError due to incompatibility spring-context.
May be additional maven profile can help but should I do this? It may be too complex.

Reply:
True, let's just stick to the latest version then